Meta/CMake/use_linker.cmake

```diff
@@ -4,6 +4,12 @@
 # SPDX-License-Identifier: BSD-2-Clause
 #
 
+# FIXME: Trying to use LLD on FreeBSD gives errors
```

Comment:
Isn't lld the system linker on FreeBSD? I would expect it's the default when nothing is set from this CMake file. Is this suggesting that -DCMAKE_LINKER_TYPE doesn't work correctly on FreeBSD? If not that would be a good thing to report to CMake's bug tracker.